pkg/cmd/webhook/webhook.go

```diff
@@ -44,6 +46,42 @@ func NewAdmissionHook() *cobra.Command {
 	o.RecommendedOptions.FeatureGate = featureGate
 
 	o.RecommendedOptions.AddFlags(cmd.Flags())
-
+	ops.AddFlags(cmd.Flags())
 	return cmd
 }
+
+// change the default QPS and Butst, so rewrite this func
+func RunAdmissionServer(o *admissionserver.AdmissionServerOptions, qps float32, burst int, stopCh <-chan struct{}) error {
+	config, err := o.Config()
+	if err != nil {
+		return err
+	}
+	config.RestConfig.QPS = qps
+	config.RestConfig.Burst = burst
+	server, err := config.Complete().New()
+	if err != nil {
+		return err
+	}
+	return server.GenericAPIServer.PrepareRun().Run(stopCh)
+}
+
+// Config contains the server (the webhook) cert and key.
+type Options struct {
+	QPS   float32
+	Burst int
```

Comment:
I would put adminssonServerOption in this Option. So all addFlags can be done in options.AddFlat.